virus/Pupavirus_apiscarab.md

```diff
@@ -0,0 +1,246 @@
+The other Apis cerana sample was from a study by the same group that involved looking at how circRNAs
+are involved in A. cerana larval response to A. apis (BioProject PRJNA560730). In this experiment, 
+their test sample had the virus while their control run did not, which is what we would expect to see. 
+Since we know for sure that the control sample had the virus, we wanted to see if it also had traces 
+of the fungus that might have gone undetected. Using STAT SRA, we found trace amounts of Ascosphaeraceae 
+and Narnaviridae in the Apis cerana SRRs (Table 2). We also found that there seems to be a corelation
+between the amount of Ascosphaeraceae and Narnavidae in the sample. 
+```
+
+Table 2: Counts of Ascosphaeraceae and Narnaviridae reads in Apis cerana treatment and control samples
+![](data/img/stat_sra_ac.png)
+
+```
+With this potential correlation in mind and due to the co-occurence of several samples with A. apis, 
+we propose that P. apiscarab uses A. apis as a vector to infect its host. The relationship between
+A. apis and P. apiscarab remains elusive, however, it is possible that P. apiscarab could be related 
+to Chalkbrood disease, although further research needs to be done.
+```
+
+### P. apiscarab is spread through horizontal transfer
+```
+ P. apiscarab was also found in five A. mellifera and one Varroa destructor samples that were 
+unrelated to A. apis (BioProject PRJNA706851). These samples were generated in a study 
+investigating honeybee viruses in China, where they identified 23 novel viruses, although none of
+them are Narnaviruses or belong to a related clade. These samples were pooled from 155 libraries 
+of honeybee and mite samples obtained from 326 honeybee colonies in various locations in China 
+between 2016 and 2019 [Li, 2023]. We ran SRA STAT on these samples to determine if A. apis (or 
+related families) were found in these samples, and no hits were found.
+Taking into consideration the wide variety of samples, nature of A. apis’ attack system on Apis
+mellifera, and the fact that A. apis is not found in these samples, we hypothesize that P. apiscarab
+is able to infect its host through horizontal transfer. In essence, since A. apis spreads between
+colonies through feeding, the virus is able to infect A. mellifera adults when they come into contact
+with A. apis. It is also likely that P. apiscarab can be spread through Varroa destructor mites 
+when they feed on honeybees.
+```
+
+```
+Finally, P. apiscarab was found in a run with Panicum virgatum (switch grass) and Tupaia belangeri
+(tree shrew). We wanted to determine if this virus could be found in a specific microbiome or if 
+these were due to contamination. To do this, we ran MGnify Search on contig 1, however no significant
+results were generated [Richardson, 2023]. Thus, the inclusion of the virus in this sample was likely
+due to some sort of contamination.
+```
+
+
+### Virus Genome {Q3}
```

Comment:
Improve title, I'd also recommend for your report to move the genome up higher and present it first. It's a great overview (include which SRR it's from) and will make all the subsequent analysis easier to understand/make sense when you talk RdRp and such.